svir/utilities/utils.py

```diff
@@ -1031,3 +1031,23 @@ def warn_scipy_missing(message_bar):
     msg = ("This functionality requires scipy. Please install it"
            " and restart QGIS to enable it.")
     log_msg(msg, level='C', message_bar=message_bar)
+
+
+def get_credentials(server):
+    """
+    Get from the QSettings the credentials to access the OpenQuake Platform
```

Comment:
I guess it's not only for the Platform

Reply:
Yes, I forgot to change the docstring, thanks